src/Components/MPO.jl

```diff
@@ -160,3 +160,53 @@ function Base.rand(rng::Random.AbstractRNG, ::Type{MPO}; n, maxdim=nothing, elty
     # TODO order might not be the best for performance
     return MPO(arrays; order=(:l, :i, :o, :r))
 end
+
+"""
+Builds a translationally invariant MPO of length L for
+    H = sum_i    c1_i * O_i
+    + sum_{i<j} c2 * O_i ⊗ O_j
+where O_i and O_j are local operators acting on sites i and j, respectively.
+
+#Arguments
+  - `L` : length of the MPO
+  - `one_body` : list of (i, O, c1)            single‐site terms
+  - `two_body` : list of (i, j, O_i, O_j, c2)  two‐site terms
+"""
+function autoMPO_periodic(L, one_body, two_body; type = ComplexF64)
```

Comment:
Mind changing the name to `autompo_periodic`? The convention is that functions are in snake_case, types and traits are in camelCase.

```suggestion
function autompo_periodic(L, one_body, two_body; type = ComplexF64)
```